cmd/controller-manager/main.go

```diff
@@ -41,3 +41,7 @@ func main() {
 		os.Exit(1) //nolint:gocritic
 	}
 }
+
+// Without init, the _ imports do not initialize before app package
```

Comment:
Could you help to link the related golang specification about this behavior? Very curious about this.

Reply:
@mars1024 i couldnt find a good doc on this, this is the only thing i could find https://golang.org/doc/effective_go#blank_import

```
For example, during its init function, the net/http/pprof package registers HTTP handlers that provide debugging information.
```

This implies that init is needed for a side effect import, though it does not clearly state it.

I've tested it extensively and it is fixing it.

Reply:
Should i re-phrase the comment 
```
// init for _ imports 
```
instead ?